Homework8/BankApplicationPractice/BankLibrary/AccountsCollectionEnumerator.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace BankLibrary
+{
+    public class AccountsCollectionEnumerator : IEnumerator<Account>
+    {
+        private readonly List<Account> _accounts;
+        private int position = -1;
+
+        public AccountsCollectionEnumerator(List<Account> accounts)
+        {
+            _accounts = accounts;
+        }
+
+        public object Current
+        {
+            get
+            {
+                if (position == -1 || position >= _accounts.Count)
+                    throw new InvalidOperationException();
```

Comment:
code style violation
```suggestion
                if (position == -1 || position >= _accounts.Count)
                {
                    throw new InvalidOperationException();
                }
```